clustered/client/build.gradle

```diff
@@ -20,6 +20,8 @@ plugins {
 }
 
 dependencies {
+  implementation "org.terracotta:terracotta-utilities-tools:$terracottaUtilitiesVersion"
```

Comment:
Please use the existing format to add dependency. Also it will be nice if it ends up with other "implementation" dependencies